src/main/webapp/data-visualization.js

```diff
@@ -294,21 +300,38 @@ function setStyle(isCountyQuery) {
     const buttonsDiv = document.getElementById('buttons');
     buttonsDiv.style.display = 'none';
   }
-
   const chartsDiv = document.getElementById('am-charts');
   chartsDiv.style.display = 'block';
   const mapsDiv = document.getElementById('map');
   mapsDiv.style.display = 'none';
 }
 
+// Changes the color of the current visualizations on the page.
+function changeColor(colorParam) {
+  const color = colorParam.value;
+  localStorage.setItem('color', color);
+  // Get variables out of cache
+  const cacheMapsData = JSON.parse(localStorage.getItem('mapsData'));
+  const cacheGeoData = JSON.parse(localStorage.getItem('geoData'));
+  const cacheAmCharts = JSON.parse(localStorage.getItem('amChartsData'));
+  const cacheLocation = localStorage.getItem('location');
+  const cacheDescription = localStorage.getItem('description');
+  // map is undefined on a state query, so check to be sure that 
+  // it is undefined before calling displayCountyGeoJson.
```

Comment:
nit: check to be sure that it isn't undefined

Reply:
updated.

Reply:
it looks the same to me? (sorry!)

Reply:
Should be fixed now but it isn't showing up on my computer so let me know if there are any more bugs!